### PR TITLE
Feature/t4 t 1142 science search

### PR DESCRIPTION
**Pull request by kb-smarterbalanced**
For story: https://smarterbalanced.atlassian.net/browse/T4T-1142

### CODE REVIEW

File: webapp/src/app/data/search/search-request.model.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export interface SearchRequestModel {
     Claim: string[];
     Domain: string[];
     DCIStrand: string[];
```
Comment (JJ-Muller) [inline webapp/src/app/data/search/search-request.model.ts:8]: Search parameter is DciStrand instead of DCIStrand

**Review by JJ-Muller — CHANGES_REQUESTED** [review]

File: webapp/src/app/search/search.component.html [hunk 2]
```diff
@@ -124,4 +124,36 @@ <h3>Content Standards</h3>
...
+    <h3>DCI Strands</h3>
+    <sbdl-filter-chipset
+      *ngIf="filters.dciStrands.length > 0"
+      [filters]="filters.dciStrands"
+      (whenChanged)="search({dciStrands: $event.join(',')}, false)"
+    ></sbdl-filter-chipset>
+  </div>
+  <div class="filter-container"
+       *ngIf="params.subjects === 'sci'">
+    <h3>PerformanceExpectations</h3>
```
Comment (JJ-Muller) [inline webapp/src/app/search/search.component.html:147]: A space is missing between Performance Expectations

**Review by JJ-Muller — CHANGES_REQUESTED** [review]

File: webapp/src/app/data/search/search-request.model.ts (hunk 1 above)
Comment (kb-smarterbalanced) [reply webapp/src/app/data/search/search-request.model.ts]: Fixed

File: webapp/src/app/search/search.component.html (hunk 2 above)
Comment (kb-smarterbalanced) [reply webapp/src/app/search/search.component.html]: This is also fixed now - thanks for the catch!

**Review by JJ-Muller — APPROVED** [review]

**Pull request merged**